helpers.sh

```diff
@@ -113,6 +113,7 @@ function set_base_lmp_version {
 
 	# 3: Find our base LMP version based on the HEAD
 	export LMP_VERSION=$(git describe --tags --abbrev=0 HEAD)
+	export LMP_VERSION_MINOR="$(git rev-list ${LMP_VERSION}..HEAD --count)"
```

Comment:
there is a typo on the commit lot "nunber"

Reply:
fixed! thanks